blocks/rich-text/index.js

```diff
@@ -212,6 +219,23 @@ export default class RichText extends Component {
 	}
 
 	onFocus() {
+		// For virtual keyboards, always scroll the focussed editor into view.
+		// Unfortunately we cannot detect virtual keyboards, so we check UA.
+		if ( /iPad|iPhone|iPod|Android/i.test( window.navigator.userAgent ) ) {
```

Comment:
Is it worth making a `isMobile` util?

Reply:
Maybe. Would be nice if there are more uses. In this specific case we're not really trying to detect "mobile" devices, but rather devices with virtual keyboards.

Reply:
Agreed. My thinking was also something like:

```js
isMobile = () => regex.test( … )
hasVirtualKeyboard = isMobile // best approximation
```